utils/csv2yaml.py

```diff
@@ -0,0 +1,176 @@
+import argparse
+import pandas as pd
+from pathlib import Path
+import yaml
+
+
+SUPPORTED_TYPES = {"text": "string", "integer": "integer", "date": "date", "float": "float",
+                   "boolean": "boolean"}
+CONVERT_TYPES = {"integer": int}
+
+class YamlDumper(yaml.Dumper):
+    def write_line_break(self, data=None):
+        super(YamlDumper, self).write_line_break(data)
+        if len(self.indents) == 2:  # This checks if we're at the top level of the document
+            super(YamlDumper, self).write_line_break()  # Insert an additional line break
```

Comment:
perhaps switch to `ruamel` (also used in dandi cli) which provides better yaml handling.